app/Nova/DuesPackage.php

```diff
@@ -156,10 +164,18 @@ public function fields(Request $request): array
                     ->onlyOnDetail(),
 
                 DateTime::make('Access Start Date', 'access_start')
+                    ->help(
```

Comment:
I would add that it's normal for this date range to be different/longer than the effective date range.

Reply:
Good call.

Reply:
Added a sentence